va/va.h

```diff
@@ -4174,6 +4174,24 @@ VAStatus vaEndPicture(
     VAContextID context
 );
 
+/**
+ * Same behavior with vaEndPicture except a sync fd list for synchronizations.
+ * if sync_num = 0 or sync_fds == NULL, the behavior should be same with vaEndPicture.
```

Comment:
Should be: "Same behaviour as" and ... "the behaviour should be the same as vaEndPicture()"